### PR TITLE
Fixes XSS in the driver test suite

### PR DESCRIPTION
**Pull request by aik099**
I've tested on all drivers except Sahi, which test suite takes too much time to run in any case.

Another PR for MinkBrowserKit driver would be sent, because it overrides fixture files that were fixed in main test suite.

Replaces #677

### CODE REVIEW

File: driver-testsuite/web-fixtures/utils.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?php
+
+function htmlspecialchars_array($data)
+{
+    if (!is_array($data)) {
+        return htmlspecialchars($data, ENT_QUOTES, 'UTF-8', false);
+    }
+
+    $escaped_data = array();
```
Comment (stof) [inline driver-testsuite/web-fixtures/utils.php]: this should use a camelCased name

File: driver-testsuite/web-fixtures/utils.php [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<?php
+
+function htmlspecialchars_array($data)
```
Comment (stof) [inline driver-testsuite/web-fixtures/utils.php]: the name is misleading IMO, as it handles more than arrays

Comment (aik099) [inline driver-testsuite/web-fixtures/utils.php]: Anything specific you have in mind?

File: driver-testsuite/web-fixtures/utils.php (hunk 1 above)
Comment (aik099) [inline driver-testsuite/web-fixtures/utils.php]: Fixed.

File: driver-testsuite/web-fixtures/utils.php (hunk 2 above)
Comment (stof) [inline driver-testsuite/web-fixtures/utils.php]: ``html_escape_value`` ?

**Comment by stof** [comment]
@aik099 I was about to release a new version of Mink, but I think this may be merged before the release.

File: driver-testsuite/web-fixtures/utils.php (hunk 2 above)
Comment (aik099) [inline driver-testsuite/web-fixtures/utils.php]: Will do.

Comment (aik099) [inline driver-testsuite/web-fixtures/utils.php]: Change pushed.

**Pull request merged**